odoo/orm/environments.py

```diff
@@ -88,11 +85,11 @@ def __new__(cls, cr, uid, context, su=False):
     # Mapping methods
     #
 
-    def __contains__(self, model_name):
+    def __contains__(self, model_name) -> bool:
         """ Test whether the given model exists. """
         return model_name in self.registry
 
-    def __getitem__(self, model_name: str) -> BaseModel:
+    def __getitem__(self, model_name: typing.Any) -> BaseModel:
```

Comment:
Please explain why `typing.Any` instead of `str`.

Reply:
More specific type defined in fields. It should have been str.